dali/imgcodec/decoders/CMakeLists.txt

```diff
@@ -13,7 +13,17 @@
 # limitations under the License.
 
 add_subdirectory(jpeg)
+add_subdirectory(nvjpeg)
 
 collect_headers(DALI_INST_HDRS PARENT_SCOPE)
 collect_sources(DALI_IMGCODEC_SRCS PARENT_SCOPE)
 collect_test_sources(DALI_IMGCODEC_TEST_SRCS PARENT_SCOPE)
+
+if (NOT BUILD_NVJPEG2K)
+    list(REMOVE_ITEM DALI_INST_HDRS "${CMAKE_CURRENT_SOURCE_DIR}/nvjpeg2k.h")
```

Comment:
how about `nvjpeg/nvjpeg2k_helper`?

Reply:
Temporarily enclosed it with `#if NVJPEG2K_ENABLED` clause, I will open follow up PR with extracting nvjpeg2k utils from nvjpeg directory, to keep this one tidy.